src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,24 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final LocalDate now = LocalDate.now();
+    private static final String DISPLACEMENT = "+02:00";
+    private static final DateTimeFormatter FORMATTER
+            = DateTimeFormatter.ofPattern("dd MMM yyyy", Locale.ENGLISH);
+    private static final DateTimeFormatter FORMATTER_ENG
```

Comment:
please use more informative names for the people who will read your code for the first time